react/src/components/blocks/Region/Sidebar/SidebarNav/SidebarNav.js

```diff
@@ -58,6 +67,20 @@ const SidebarNav = (props) => {
           </Accordion>
           <CreateChannel show={showModal} handleModalClose={handleModalClose}/>
         </div>
+      </ContextMenuTrigger>
+
+      <ContextMenu className={styles.menu} id="select_options">
+          <MenuItem onClick={handleClick} data={{item: ""}}
+          className={`${styles.menuItem} ${styles.menuItemFirst}`}>
+              {formatMessage({ id: 'create_sidebar_section' })}
+          </MenuItem>
+          <MenuItem onClick={handleClick} data={{item: ""}} className={styles.menuItem}>
+                {formatMessage({ id: 'browse_channels' })}
+          </MenuItem>
+          <MenuItem onClick={handleChannelAdd} data={{item: "Create a Channel"}} className={styles.menuItem}>
```

Comment:
Create a Channel - localise

Reply:
No need of localize and i have updated the code